Resources/Prototypes/Catalog/Cargo/cargo_armory.yml

```diff
@@ -24,7 +24,7 @@
     sprite: Objects/Specific/Medical/implanter.rsi
     state: implanter0
   product: CrateTrackingImplants
-  cost: 1000
+  cost: 4000
```

Comment:
```suggestion
  cost: 4000 # DeltaV - was 1000
```

Reply:
You're totally right, forgot the comment (that is a bad trend i am having), sorry.